lib/OpenQA/WebAPI/Controller/Running.pm

```diff
@@ -115,9 +114,9 @@ sub livelog {
     # logfile and send them to the client, plus a utility function to
     # close the connection if anything goes wrong.
     my $id;
-    my $close = sub {
+    local *doclose = sub {
```

Comment:
This is how perlcritic/bestpractices and the documentation recommend to do it: http://perldoc.perl.org/perlref.html#Function-Templates

However, looking at it again, I am not sure that `local *sub = sub {}` is better than `my $sub = sub {}` except that you can call it as `sub()` rather than `sub->()` inside the enclosing lexical scope. The `local` keyword dynamically scopes variables so I think that it is best avoided, but it is kind of strange that using `my $sub = sub {}` is not even mentioned in the documentation I have seen (and can remember).